pkg/assembler/graphql/schema/package.graphql

```diff
@@ -133,6 +133,9 @@ Each field matches a qualifier from pURL. Use null to match on all values at
 that level. For example, to get all packages in GUAC backend, use a PkgSpec
 where every field is null.
 
+The id field can be used to match on a specific node in the trie to match packageTypeID, 
```

Comment:
will have to run `make generate` for this to update the auto generated code.

Reply:
Just pushed those changes